blocks/header/header.css

```diff
@@ -246,7 +251,7 @@
   display: flex;
   cursor: pointer;
   flex-direction: column;
-  justify-content: center; 
+  justify-content: center;
```

Comment:
too many whitespace changes in this file.

Reply:
will try to remove these changes